js/config/app.config.js

```diff
@@ -1,7 +1,7 @@
-define(['optional!config-local'], function (localConfig) {
+define(['optional!config/local.config'], function (localConfig) {
```

Comment:
Since this is not a major release, pls add fallback for old `config-local.js` file